### PR TITLE
[#40] Remove duplication of answers

### LINKED ISSUE #40: Questions: Remove duplication from answers
The answer can contain just the `a`, `b`, `c` etc.  like so:
```
<panel header=":lock::key: What are the advantages of using use cases (the textual form) for requirements modelling?">
<question>

What are the advantages of using use cases (the textual form) for requirements modelling?

- [ ] a. They can be fairly detailed but still natural enough for users for users to understand and give feedback.
- [ ] b. The UI-independent nature of use case specification allows the system designers more freedom to decide how a functionality is provided to a user.
- [ ] c. Extensions encourage us to consider all situations a software product might face during its operations.
- [ ] d. They encourage us to identify and optimize the typical scenario of usage over exceptional usage scenarios.
- [ ] e. All of the above.
- [ ] f. First three only.

<div slot="answer">

e

</div>
</question>
</panel>
```

### PR DESCRIPTION
**Pull request by jamessspanggg**
Fixes #40 and #72

Removed duplication of answers. Also fixed some obvious errors along the way.

### CODE REVIEW

**Comment by jamessspanggg** [comment]
@damithc I went through the entire textbook and checked for any duplications, while removing some obvious errors such as using radio buttons when answer consists of multiple choices etc.

File: principles/dependencyInversionPrinciple/q-tick-dip.md [hunk 1]
```diff
@@ -10,10 +10,10 @@ Which of these statements is true about the Dependency Inversion Principle.
...
 <div slot="answer">
 
-- [x] a. It can complicate the design/implementation by introducing extra abstractions, but it has some benefits.
-- [ ] b. It is often used during testing, to replace dependencies with mocks.
-- [ ] c. It reduces dependencies in a design.
-- [ ] d. It advocates making higher level classes to depend on lower level classes.
+- [x] a.
+- [ ] b.
+- [ ] c.
+- [ ] d.
```
Comment (damithc) [inline principles/dependencyInversionPrinciple/q-tick-dip.md:16]: Just `a` is enough I guess.

File: principles/review/q-radio-oddOneOut.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Pick the odd one out.
 
 <div slot="answer">
 
-- (x) a. Separation of Concerns Principle.
+- (x) a
```
Comment (damithc) [inline principles/review/q-radio-oddOneOut.md:21]: No need for `(x)`

**Comment by jamessspanggg** [comment]
> Couple of nits; might apply to other places too.

Done.

**Pull request merged**